websites/P/Pixelmon Mod/presence.ts

```diff
@@ -0,0 +1,65 @@
+const presence = new Presence({
+		clientId: "976435781486911509"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		title =
+			document.querySelector(
+				"#page-body > main > div > div.col-md-9 > div.side-segment > h3 > a"
+			) ??
+			document.querySelector(
+				"#page-body > main > div > div.col-md-9 > div.side-segment > h3"
+			),
+		search = document.querySelector<HTMLInputElement>("#searchInput");
+	if (pathname.includes("news.php") || pathname === "/")
+		presenceData.details = "Homepage";
+	else if (pathname.includes("/wiki/") || pathname.includes("/w/index.php?")) {
+		const firstChild = document.querySelector(
```

Comment:
Surely this isn't that robust

Reply:
Beep boop

Reply:
it says it isnt outdated but it isnt using the latest commit.